<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Whiteboard</class>
 <widget class="QMainWindow" name="Whiteboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Whiteboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveAs"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionPen_Color"/>
    <addaction name="actionPen_Width"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Screen"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionEraser"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionPen_Color">
   <property name="text">
    <string>Pen Color</string>
   </property>
  </action>
  <action name="actionPen_Width">
   <property name="text">
    <string>Pen Width</string>
   </property>
  </action>
  <action name="actionClear_Screen">
   <property name="text">
    <string>Clear Screen</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
